syntax = "proto2";

package message;

message FunctionCall {
    required string user = 1;
    required string function = 2;
    required string resultKey = 3;

    optional bool success = 4;
    optional bool isAsync = 5;

    optional bytes inputData = 6;
    optional bytes outputData = 7;
}

